lib/ansible/modules/monitoring/grafana_dashboard.py

```diff
@@ -195,6 +200,22 @@ def get_grafana_version(module, grafana_url, headers):
     return int(grafana_version)
 
 
+def grafana_folder_exists(module, grafana_url, grafana_version, id, headers):
+    if grafana_version < 5:
```

Comment:
I think that this behaviour deserves to be documented in the module doc so that people won't loose time trying to use it.